### PR TITLE
Docs(web-react, web): Add Text demo #DS-911

### PR DESCRIPTION
**Pull request by pavelklibani**
## Description

Text demos are now same in all packages

### Issue reference

[Demo: Text má ve všech technologiích stejná dema](https://jira.lmc.cz/browse/DS-911)

### CODE REVIEW

File: packages/web/src/scss/helpers/text/index.html
Comment (literat) [inline packages/web/src/scss/helpers/text/index.html:1]: In this case I would like to let @adamkudrna decide whether this example should change.

**Review by literat — APPROVED** [review]
LGTM

File: packages/web/src/scss/helpers/text/index.html
Comment (crishpeen) [reply packages/web/src/scss/helpers/text/index.html:1]: IMHO we shouldn't change this. We don't have a Text component in the Web package, so we shouldn't compare the demos. It is the same as we won't do this for the Field subcomponents, it doesn't make sense. 

Also, we need to keep the truncated demo somewhere. 

I don't mind extending the text helper page, but IMHO we shouldn't try to make them the same with other packages.

File: packages/web-react/src/components/Text/demo/index.tsx [hunk 1]
```diff
@@ -2,31 +2,21 @@
 /* eslint-disable import/no-extraneous-dependencies, import/extensions, import/no-unresolved */
```
Comment (crishpeen) [inline packages/web-react/src/components/Text/demo/index.tsx:2]: do we need this line?

File: packages/web/src/scss/helpers/text/index.html
Comment (pavelklibani) [reply packages/web/src/scss/helpers/text/index.html:1]: I put the previous code back and added a new structure with docs-Stack for text helper demo

File: packages/web-react/src/components/Text/demo/index.tsx (hunk 1 above)
Comment (pavelklibani) [reply packages/web-react/src/components/Text/demo/index.tsx]: we don't, line removed

**Review by crishpeen — APPROVED** [review]

**Pull request merged**